flytekit/clis/flyte_cli/main.py

```diff
@@ -485,6 +487,7 @@ def _render_schedule_expr(lp):
 _output_location_prefix_option = _click.option(
     "-o", "--output-location-prefix", help="Custom output location prefix for offloaded types (files/schemas)"
 )
+_output_config_format = _click.option("-o", "--format", type=str, default="ini", help="Output config format (ini/yaml)")
```

Comment:
can we use click.Choice here? https://click.palletsprojects.com/en/8.0.x/options/#choice-options

Reply:
sure, thnaks.